source/tests/infer/gen_dpa3.py

```
@@ -0,0 +1,208 @@
+                "build*",
+                "op",
+                "pt",
+                "libdeepmd_op_pt.so",
+            ),
+        ]:
+            libs = glob.glob(pattern)
+            if libs:
+                torch.ops.load_library(libs[0])
+                break
+except Exception:
+    pass
```

Ruff (0.15.4)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)